backend/tracim_backend/lib/core/workspace.py

```diff
@@ -83,6 +84,11 @@ def create_workspace(
         agenda_enabled: bool = True,
         save_now: bool = False,
     ) -> Workspace:
+        from tracim_backend.lib.core.user import UserApi
```

Comment:
When import is not at top of file for a specific reason, it is preferable to indicate why in a note.

Reply:
explained.